src/public/js/script.js

```diff
@@ -176,21 +184,46 @@ function handleRadioClick() {
   }
 }
 
-function showAlert(type, message) {
-  const alertDiv = document.createElement('div')
-  alertDiv.className = `alert alert-${type} alert-dismissible fade show`
-  alertDiv.textContent = message
-
-  const container = document.querySelector('.container')
-  container.appendChild(alertDiv)
-
-  setTimeout(function () {
-    alertDiv.remove()
-  }, 3000)
-}
-
 const radioButtons = document.querySelectorAll('input[name="example"]')
 
 radioButtons.forEach((radio) => {
   radio.addEventListener('click', handleRadioClick)
 })
+const addButton = document.getElementById("add");
+
+
+function removeInputFields(event) {
+  const parentDiv = event.target.closest('.delivery-address');
+  parentDiv.remove();
+}
+addButton.addEventListener("click", function() {
+  const newInputFields = document.createElement("div");
+  newInputFields.innerHTML = `
+<div class="row delivery-address">
+
+            <div class="col-md-2">
+                <div class="mb-3">
+                    <label for="fromCity" class="form-label">From (City,State)</label>
+                    <input type="text" class="form-control" id="fromCity">
+                </div>
+            </div>
+            <div class="col-md-2">
+                <div class="mb-3">
+                    <label for="toCity" class="form-label">To (City, State)</label>
+                    <input type="text" class="form-control" id="toCity">
+                </div>
+            </div>
+            <div class="col-md-2" style='margin-top: 33px'>
+                <div class="mb-3">
+                    <button id="remove" style='border: 2px solid #6699cc; font-weight: bold; width:40px' type="button" class="btn btn-light">-</button>
+                </div>
+            </div>
+        </div>
+            `;
+  document.querySelector(".delivery-address").appendChild(newInputFields);
+});
+document.addEventListener("click", function(event) {
+  if (event.target.id === "remove") {
+    removeInputFields(event);
+  }
+});
```

Comment:
Why we're adding event listener to whole document? Id should be unique accross document

Reply:
done